app/services/workflow_reporter.rb

```diff
@@ -5,10 +5,11 @@ class WorkflowReporter
   PRESERVATIONAUDITWF = 'preservationAuditWF'
   NO_WORKFLOW_HOOKUP = 'no workflow hookup - assume you are in test or dev environment'
   COMPLETED = 'completed'
+  MISSING_WF_REGEX = /^Failed .*#{Settings.workflow_services_url}.*preservationAuditWF.* (HTTP status 404)/.freeze
```

Comment:
this seemed like the best way to identify the error messages pertaining to "workflow does not yet exist" concept.